torch/_torch_docs.py

```diff
@@ -610,6 +610,105 @@ def merge_dicts(*dicts):
     True
 """)
 
+add_docstr(torch.all,
+           r"""
+all(input) -> Tensor
+
+Tests if all elements in :attr:`input` evaluate to `True`.
+
+Example::
+
+    >>> a = torch.rand(1, 2).bool()
+    >>> a
+    tensor([[False, True]], dtype=torch.bool)
+    >>> torch.all(a)
+    tensor(False, dtype=torch.bool)
+    >>> a = torch.arange(0, 3)
+    >>> a
+    tensor([0, 1, 2])
+    >>> torch.all(a)
+    tensor(False)
+
+.. function:: all(input, dim, keepdim=False, *, out=None) -> Tensor
+
+For each row of :attr:`input` in the given dimension :attr:`dim`
+returns True if all elements in the row evaluate to `True`, `False` otherwise.
```

Comment:
"\`True\`, \`False\` otherwise" -> "\`True\` and \`False\` otherwise."

Reply:
(also the first True on this line needs backticks for consistency)

Reply:
Done. Thanks!